controllers/admin/AdminMollieAjaxController.php

```diff
@@ -18,6 +19,9 @@ public function postProcess()
             case 'resendPaymentMail':
```

Comment:
Case statement argument strings could be added to the private or public constants (depends on if they will be used by other classes) for better readability and reusability. This is just an insight.